### ISSUE TITLE
Set/get Run As user

### ISSUE THREAD

**Issue by jlfsjunior** [rstudio/connectapi#57]
Thanks a lot for the effort on this package! It does a lot of things that I wanted to be able to do without the UI.

I came because of the set/get vanity URL that works quite nicely AFAIK. It may be greedy to ask for more, but would it be possible to have a similar pair of functions for the Run As user? I saw that they have already `audit_` function and I imagine the set/get is a very different story, but would be nice to have it.

Thanks!

**Comment by colearendt** [comment]
Thanks for reporting this! There are a few ways to "retrieve" the RunAs user today, but definitely not a public / documented way to "set" the `RunAs` user, unfortunately.

If you ever have the chance to share feedback on the types of workflows that being able to programmatically "drive" Connect unlocks, we would definitely love to hear over on [RStudio Community](https://community.rstudio.com/c/r-admin/rstudio-connect/)!! Customer use cases help us prioritize new work on the product, and it may help others' imaginations, as well!

One way to retrieve today:
```
client <- connect()
myapp <- content_item(client, "myguid")
myapp$get_content_remote()$run_as
```

Unfortunately, the attribute is read only, so `myapp$update()` won't change the `run_as` user. If it is `NULL`, that just means that it has not been customized. `set_run_as` or some such will definitely be a nice addition!

**Comment by jlfsjunior** [comment]
Just for completeness, let me share how I intend to use this functionality here. 

In the current state publishers are not allowed to change vanity urls or run as users, therefore they are required to contact admins and ask us to do so. What I want to do is to make these changes available to a selected list of publishers for only the content they own, so I was thinking about an API that uses this package on the backend with a specific admin api key to accomplish this goal.

An extra logic is required to validate vanity urls and RunAs users, for instance to check if the publisher has permission to use user X or to enable her to have an url that starts with '/projectX' or '/userY'. But as long as admins can set those parameters using the package, the rest is just a few lines of code from my side.

**Comment by colearendt** [comment]
Thanks so much for the background here @jlfsjunior !! I just implemented a first pass at this in #61 . 

As I mentioned before, you can _get_ the `run_as` user with the code above. Now you can also _set_ the run_as user with:

```
set_run_as(myapp, "my-username")
```

Please let us know if you run into any issues with this workflow, or as you have more feedback on the API / package and how it can help resolve your problems!

**Comment by jlfsjunior** [comment]
Thanks, @colearendt! I will test it out!
